CHANGELOG.md

```diff
@@ -4,6 +4,244 @@ All notable changes to this project will be documented in this file.
+- Allow attaching and clearing scripts on multiple nodes at once.
+- Default values are no longer saved in scene and resource files.
+- The selection rectangle of 2D nodes is now hidden when not pertinent (no more rectangle for collision shapes).
+- SSE2 is now enabled in libsquish, resulting in improved S3TC encoding performance.
+- Tangent and binormal coordinates are now more consistent across mesh types (primitive/imported), resulting in more predictable normal map and depth map appearance.
+- Better defaults for 3D scenes.
+  - The default procedural sky now has a more neutral blue tone.
+  - The default SpatialMaterial now has a roughness value of 1 and metallic value of 0.
+  - The fallback material now uses the same values as the default SpatialMaterial.
+- Text editor themes are now sorted alphabetically in the selection dropdown.
+- The 3D manipulator gizmo now has a smoother, more detailed appearance.
+- The 3D viewport menu button now has a background to make it easier to read.
+- Controls inside containers are no longer movable or resizable but can still be selected.
+- The `is` GDScript keyword can now be used to compare a value against built-in types.
+- Named enums in GDScript no longer create script constants.
+  - This means `enum Name { VALUE }` must now be accessed with `Name.VALUE` instead of `VALUE`.
+- Shadowing variables from parent scopes is no longer allowed in GDScript.
+- Function parameters' default values can no longer depend on other parameters in GDScript.
+- Indentation guides are now displayed in a more subtle way in the script editor.
+  - Indentation guides are now displayed when indenting using spaces.
+- Multi-line strings are now highlighted as strings rather than as comments in the script editor.
+  - This is because GDScript does not officially support multiline comments.
+- Increased the script editor's line spacing (4 pixels → 6 pixels).
+- Increased the caret width in the script editor (1 pixel → 2 pixels).
+- The project manager window is now resized to match the editor scale.
+- The asset library now makes use of threading, making loading more responsive.
+- Line spacing in the script editor, underlines and caret widths are now resized to match the editor scale.
+- Replaced editor icons for checkboxes and radio buttons with simpler designs.
+- Tweaked the editor's success, error, and warning text colors for better readability and consistency.
+- **Android:** Custom permissions are now stored in an array and their amount is no longer limited to 20.
+  - Custom permissions will have to be redefined in projects imported from older versions.
+- **Android:** Provide error details when an in-app purchase fails.
+- **Linux:** `OS.alert()` now uses Zenity or KDialog if available instead of xmessage.
+- **Mono:** Display stack traces for inner exceptions.
+- **Mono:** Bundle `mscorlib.dll` with Godot to improve portability.
+
+### Removed
+
+- Support for 32-bit and fat macOS binaries.
```

Comment:
Probably it's better to use something like:
```
**macOS:** Support for 32-bit and fat binaries.
```
Otherwise some people will read it as "Removed ALL 32-bit support" and "Removed FAT macOS binaries".